### PR TITLE
Add new api version for SecurityAlertPolicies for Orcas

### PR DESCRIPTION
**Pull request by ayeletshpigelman**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [ ] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

**Comment by akning-ms** [comment]
Hi @ayeletshpigelman, pls fix Lint validation failure and let us know if it is ready for review

**Comment by ayeletshpigelman** [comment]
@akning-ms regarding the Lint-diff error it should be suppressed. the resource provider name is: Microsoft.DBforMySQL
regarding the breaking changes it is by design - this why it is under a new api version

**Comment by ayeletshpigelman** [comment]
@akning-ms  I have removed version 2017-12-01 to a new file instead of mysql.json to avoid conflict. please review the changes

**Comment by akning-ms** [comment]
> @akning-ms regarding the Lint-diff error it should be suppressed. the resource provider name is: Microsoft.DBforMySQL
> regarding the breaking changes it is by design - this why it is under a new api version

Ok, got it, I have added arm for review. could you remove draft tag for PR?

**Comment by ayeletshpigelman** [comment]
@akning-ms: Hi can someone review this PR please?
regarding the Lint-diff error it should be suppressed. the resource provider name is: Microsoft.DBforMySQL
regarding the breaking changes it is by design - this why it is under a new api version. Therefore I had to move SecurityAlertPolicies APIs to a separate file.
please help me advance this PR.
thanks

**Review by QingqingYuan2 — APPROVED** [review]

**Comment by chiragg4u** [comment]
There are breaking changes, Please follow Azure Breaking changes approval request form as defined @ [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes)

**Comment by ayeletshpigelman** [comment]
@chiragg4u  and @akning-ms: these are no breaking changes. I have moved the securityAlertPolicies old version to a separated file, so I can add another version. how can wee suppress this error? and also regarding the Lint error -  the resource provider name is: Microsoft.DBforMySQL, Microsoft.DBforPostgreSQL, Microsoft.DBforMariaDB. why is it forcing a Pascal case?

**Comment by akning-ms** [comment]
> @chiragg4u and @akning-ms: these are no breaking changes. I have moved the securityAlertPolicies old version to a separated file, so I can add another version. how can wee suppress this error? and also regarding the Lint error - the resource provider name is: Microsoft.DBforMySQL, Microsoft.DBforPostgreSQL, Microsoft.DBforMariaDB. why is it forcing a Pascal case?

@ayeletshpigelman For breaking change, yes, agree on this is not really breaking change due to tools's limitation.  for lint failure, pls follow this lint to suppress it. https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process?anchor=obtaining-suppression

**Comment by pilor** [comment]
No API changes, just moving specs around.  No ARM review needed

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ServerSecurityAlertPolicyState",
+            "modelAsString": false
+          }
+        },
+        "creationTime": {
```
Comment (pilor) [inline specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json:199]: created/updated metadata should use the common systemMetadata contract for any new APIs:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

**Review by pilor — CHANGES_REQUESTED** [review]

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json (hunk 1 above)
Comment (ayeletshpigelman) [reply specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json:199]: @pilor - done

**Comment by akning-ms** [comment]
@jianyexi can you double confirm this lint failure? looks it is different result from staging version. https://dev.azure.com/azure-sdk/public/_build/results?buildId=449315&view=logs&j=688669d0-441c-57c3-cf6d-f89a22ccfa5d&t=923b5a90-e462-5994-c166-86509c6bf5f2

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json (hunk 1 above)
Comment (pilor) [reply specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json:199]: The creationTime property should be removed wherever it exists in the new api-versions

**Review by pilor — CHANGES_REQUESTED** [review]

**Comment by jianyexi** [comment]
> @jianyexi can you double confirm this lint failure? looks it is different result from staging version. https://dev.azure.com/azure-sdk/public/_build/results?buildId=449315&view=logs&j=688669d0-441c-57c3-cf6d-f89a22ccfa5d&t=923b5a90-e462-5994-c166-86509c6bf5f2

The result of staging version is correct , actually there is  a bug-fix in staging version for the rule: XmsEnumNameUnique

File: specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json (hunk 1 above)
Comment (ayeletshpigelman) [reply specification/mariadb/resource-manager/Microsoft.DBforMariaDB/stable/2020-01-01/ServerSecurityAlertPolicies.json]: @pilor - done

**Review by pilor — APPROVED** [review]

**Pull request merged**